modules/mcuboot/boot/zephyr/Kconfig

```diff
@@ -15,8 +15,14 @@ partition-size=0x1e000
 source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_config"
 
 partition=MCUBOOT_PAD
-partition-size=0x200
+if SOC_PLATFORM_NRF54L
+	partition-size=0x800
```

Comment:
Note for the future: this could probably be taken from the `CONFIG_ROM_START_OFFSET`